app/src/core/design/components/userComponents/UtilComponents/VideoProperties.js

```diff
@@ -0,0 +1,225 @@
+import { TextField, Box, Button, InputAdornment } from "@material-ui/core";
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+import { CustomAccordion } from "./Accordion";
+import { IconButton } from "@material-ui/core";
+import { Tooltip } from "@material-ui/core";
+import YouTubeIcon from "@material-ui/icons/YouTube";
+import exportImageUrl from "../../../../api/exportImageUrl";
+import CheckCircleOutlineOutlinedIcon from "@material-ui/icons/CheckCircleOutlineOutlined";
+import ReplayOutlinedIcon from "@material-ui/icons/ReplayOutlined";
+
+export function MediaAccordion({ props, setProp, src, thumbnailSrc, type, defaultThumbnail }) {
+    if (!String.prototype.format) {
+        String.prototype.format = function() {
+            var args = arguments;
+            return this.replace(/{(\d+)}/g, function(match, number) {
+                return typeof args[number] != "undefined" ? args[number] : match;
+            });
+        };
+    }
+    let thumbnailSrcValue = "",
+        tempThumbnailSrcValue = "",
+        publicId = "",
+        width = 360,
```

Comment:
we can create constants for these in VideoSettings. 
const DEFAULT_VIDEO_WITH = 360